File: 백준/Bronze/2443. 별 찍기 － 6/별 찍기 － 6.swift
import Foundation

var n = Int(readLine()!)!

for i in (0..<n).reversed() {
    var answer = ""
    for _ in 0..<(n-1-i) { answer += " " }
    for _ in 0..<(2*i+1) { answer += "*" }
    print(answer)
}

File: 백준/Bronze/2443. 별 찍기 － 6/별 찍기 － 6.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());

        for(int i=n-1; i>-1; i--) {
            String answer = "";
            for(int j=1; j<(n-i); j++) answer += " ";
            for(int k=0; k<(2*i+1); k++) answer += "*";
            System.out.println(answer);
        }
    }
}